### PR TITLE
Feature: default image

### PR DESCRIPTION
**Pull request by khcr**
- Added a default image Bitmap to the method downloadImage
- Added dependency injection to the FileStore and adapted the views and tests accordingly
- Increases coverage of FileStore

### CODE REVIEW

File: app/src/main/java/com/ncnf/firebase/modules/FirebaseStorageModule.java [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+import dagger.hilt.android.components.ActivityComponent;
+
+@Module
+@InstallIn(ActivityComponent.class)
+public class FirebaseStorageModule {
+
+    @Provides
+    public static FirebaseStorage getFirebaseStorage() {
+        return FirebaseStorage.getInstance();
+    }
```
Comment (Jnf14) [inline app/src/main/java/com/ncnf/firebase/modules/FirebaseStorageModule.java:19]: Nice to have new modules for a clean implementation !

File: app/src/main/java/com/ncnf/storage/FileStore.java [hunk 2]
```diff
@@ -47,24 +56,34 @@ public FileStore(String directory, String filename) {
...
-    public void downloadImage(ImageView view) {
+    public void downloadImage(ImageView view, Bitmap defaultImage) {
+        requiresPath();
+
         download().thenAccept(data -> {
             Bitmap bitmap = BitmapFactory.decodeByteArray(data, 0, data.length);
             view.setImageBitmap(bitmap);
+        }).exceptionally(e -> {
+            view.setImageBitmap(defaultImage);
+            return null;
```
Comment (Jnf14) [inline app/src/main/java/com/ncnf/storage/FileStore.java:67]: Nice ! Gg

**Review by Jnf14 — APPROVED** [review]
Thanks for this PR @khcr ! This will ease the use of the FileStore module !

File: app/src/androidTest/java/com/ncnf/socialObject/create/SocialObjectCreateActivityTest.java [hunk 3]
```diff
@@ -58,20 +61,16 @@
     @BindValue
```
Comment (HyPn0TiiK) [inline app/src/androidTest/java/com/ncnf/socialObject/create/SocialObjectCreateActivityTest.java:61]: What is this class name?

**Review by HyPn0TiiK — APPROVED** [review]
Clean and efficient as usual

**Pull request merged**